### DISCUSSION TITLE
workflow for printing

### DISCUSSION THREAD

**Discussion by eolmapper** [OpenOrienteering/mapper#1783] [Questions & Answers]
What is for you the right workflow to print a map (make with OOM), only with open source software?

Questions:
-How to do the layout and the integration of the logos?
_Many printer worker, work with PDF format. How do you genérate vectorial pdf  of the map and course planning symbols, with overprinting effect (not only the course planning symbols) or without (purple under black...)?

**Comment by dg0yt** [comment]
> Many printer worker, work with PDF format. How do you genérate vectorial pdf of the map and course planning symbols, with overprinting effect (not only the course planning symbols) or without (purple under black...)?

OpenOrienteering Mapper can't help you much with that at the moment, except for OCD and PDF export. I don't know too much about Purple Pen's capabilities in this regard. 

The "purple under black" approach to course setting is nearly impossible with independent software. Map creation software (e.g. Mapper) and course planning software (e.g. Purple Pen) would need to agree on how to separate the map objects below the course planning symbols from the map objects above, and find ways to implement this. However, with PDF handling provided by third-party software (e.g. Qt), this might be hard. Adding marginalia and logos with other PDF software won't make it easier.

In the end, we will need to implement course setting in Mapper.

**Comment by eolmapper** [comment]
I dont think Mapper needs a course setting module.
Générate vectorial pdf of the map and course planning symbols, is probably in the perimeter of purplepen and it doesn't need so much to do it right.
For this we have tu push up for this implémentation of Purplepen:
https://purplepen.uservoice.com/forums/13891-general/suggestions/40015369-be-able-to-use-overprint-effect-for-color-marked
And concerning purple under black..., for the moment I do an export with purplepen in .omap and I put it back in the map file in OOM.
Its working but it is not practical.

But regarding the layout and integrate logos I think that's the responsability of mapper.
The drawing softwares, have a module for that: Qgis, autocad ...
Layout it's an integral part of the map, and it's sad to make the map with OOM and have to use ocad to finish the job.
For the moment I can do that with OOM but it's DIY and there are a lot of rough edges.

**Comment by ollesmaps** [comment]
The overprint effect is not fully supported by Map Commission IOF. They prefer "purple under black".
"_Export with Purple Pen in .omap and put it back in the map file in OOM_ (and then putting purple under black)." is an interesting idea.

**Comment by eolmapper** [comment]
yes, they prefer "purple under black ... and brown" but no course setting software does that yet.
I am convinced in some cases that overprinting effect is a better choice.
